core/src/main/java/org/pentaho/di/core/plugins/BasePluginType.java

```diff
@@ -672,6 +675,8 @@ protected URLClassLoader createUrlClassLoader( URL jarFileUrl, ClassLoader class
 
   protected abstract String extractDocumentationUrl( java.lang.annotation.Annotation annotation );
 
+  protected abstract String extractSuggestedStep( java.lang.annotation.Annotation annotation );
```

Comment:
Any reason why this can't be a default implementation so we don't have to override in every plugin that doesn't need it?